oeps/oep-0017-bp-feature-toggles.rst

```diff
@@ -636,9 +654,13 @@ Specifying toggle configuration in environment variables or command-line argumen
 Configuration Files
 -------------------
 
-Storing toggle configuration in separate files allows the configuration to be decoupled from the code and allows different deployments to override values.  However, any change to the configuration requires a redeploy of the application.
+Storing toggle configuration in separate files allows the configuration to be decoupled from the code and allows different deployments to override values. However, any change to the configuration requires a redeploy of the application (see note with update).
+
+.. note::
 
-Many features in the edX platform use `JSON Configuration files`_ to store their settings, including toggle configuration. It is recommended that features instead use a more dynamically configurable alternative such as `Configuration Models`_ or Feature Toggles, unless (1) the setting is security-sensitive or (2) is guaranteed to not need to change for a given open edX deployment.
+    **UPDATE:** Since first written, edX has implemented immediate updates to configuration changes that don't require full deployment. We have thus implemented some feature toggles based on Django settings. See `how to choose the right toggle class`_ for details.
```

Comment:
**Note to Reviewer:** I didn't really understand the rest of this section, so I did a soft update here.  This could always be iterated upon later.